main.tf

```
@@ -234,6 +242,8 @@ module "alb" {
     prefix  = var.alb_log_location_prefix
   }
 
+  ip_address_type = var.enable_ipv6 ? "dualstack" : "ipv4"
```

ya lets just make this
```suggestion
  ip_address_type = var.alb_ip_address_type
```

And the variable default will be `"ipv4"` like the ALB module does